views/index.ejs

```diff
@@ -2,18 +2,15 @@
 <html>
   <head>
       <% include ./partials/head %>
+      <script src="http://code.jquery.com/jquery-latest.min.js" type="text/javascript"></script>
```

Comment:
Is this actually being used anywhere aside from index.js? If not, please remove this as the ajax calls will soon be replaced with socket.io calls. (Also external cdn usage isn't handy here)